### ISSUE TITLE
Removes Jaeger Agent from Jaeger Hotrod Demo

### ISSUE THREAD

**Issue by linghengqian** [opensearch-project/data-prepper#4923]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. It would be nice to have [...]
- See https://github.com/opensearch-project/data-prepper/issues/2273#issuecomment-1464092972 .
- Starting from https://github.com/jaegertracing/jaeger/pull/4187, Jaeger HotROD no longer uses `jaegertracing/jaeger-agent` to receive `jaegertracing/example-hotrod` data, but directly uses `otel/opentelemetry-collector-contrib`. According to https://www.jaegertracing.io/docs/1.60/deployment/#agent , jaegertracing/jaeger-agent is considered a deprecated component. It does not make any sense.
- According to https://opentelemetry.io/docs/collector/installation/ and https://opentelemetry.io/docs/collector/deployment/agent/ , removing the use of the Jaeger Agent is very simple.
```yaml
services:
  opensearch-node1:
    hostname: node-0.example.com
    image: opensearchproject/opensearch:2.16.0
    environment:
      discovery.type: single-node
      OPENSEARCH_INITIAL_ADMIN_PASSWORD: opensearchNode1Test
    volumes:
      - opensearch-config1:/usr/share/opensearch/config/
    expose:
      - 9200
      - 9600
  opensearch-dashboards:
    image: opensearchproject/opensearch-dashboards:2.16.0
    ports:
      - "14321:5601"
    environment:
      OPENSEARCH_HOSTS: '["https://opensearch-node1:9200"]'
    depends_on:
      - opensearch-node1
  data-prepper:
    restart: always
    image: opensearchproject/data-prepper:2.9.0
    volumes:
      - ./pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml
      - ./data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml
      - opensearch-config1:/usr/share/opensearch-test/:ro
    expose:
      - 21890
    depends_on:
      - opensearch-node1
  otel-collector:
    image: otel/opentelemetry-collector-contrib:0.108.0
    volumes:
      - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
    expose:
      - 1888
      - 8888
      - 8889
      - 13133
      - 4317
      - 4318
      - 55679
    depends_on:
      - data-prepper
  jaeger-hot-rod:
    image: jaegertracing/example-hotrod:1.60.0
    command: [ "all" ]
    environment:
      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel-collector:4318
    ports:
      - "8080:8080"
    expose:
      - 8083
    depends_on:
      - otel-collector
volumes:
  opensearch-config1:

```

**Describe the solution you'd like**

- Removes Jaeger Agent from Jaeger Hotrod Demo.

**Describe alternatives you've considered (Optional)**

**Additional context**

- Null.

**Comment by dlvenable** [comment]
We can close this once #4921 is merged.

**Comment by linghengqian** [comment]
> We can close this once #4921 is merged.

#4921 does not actually remove the dependency on jaeger agent. Should I update my PR?

**Comment by linghengqian** [comment]
- Since no one responded, I opened #4972 .
